resolvers/webpack/index.js

```diff
@@ -146,15 +159,18 @@ function getResolveSync(configPath, webpackConfig) {
   return cached.value
 }
 
-function createResolveSync(configPath, webpackConfig) {
+function createResolveSync(configPath, webpackConfig, cwd) {
   var webpackRequire
     , basedir = null
 
   if (typeof configPath === 'string') {
-    basedir = path.dirname(configPath)
+    // This can be changed via the settings passed in when defining the resolver
+    basedir = cwd || configPath
```

Comment:
Should be `basedir = cwd || path.dirname(configPath)`.
The `basedir` option of `resolve.sync` is expected to be a directory path.

It will break in `resolve` v2 because of this commit https://github.com/browserify/resolve/commit/698a3e179dc0a3c3869e62669382bc489c31d333

Reply:
Should be `basedir = cwd || path.dirname(configPath)`.
The `basedir` option of `resolve.sync` is expected to be a directory path.